packages/components/addon/components/hds/app-footer/copyright.hbs

```diff
@@ -0,0 +1,14 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{! <Hds::AppFooter::Item class="hds-app-footer__list-item--copyright" ...attributes>
+  <FlightIcon @name="hashicorp" />
+  <Hds::Text::Body @tag="span" @size="100" @color="faint">© 2023 HashiCorp</Hds::Text::Body>
+</Hds::AppFooter::Item> }}
+
+<div class="hds-app-footer__copyright" ...attributes>
```

Comment:
once things are more settled, we can understand/decide if there some further optimization you can do here, for now is OK